tests/common/dualtor/mux_simulator_control.py

```diff
@@ -510,6 +511,8 @@ def toggle_all_simulator_ports_to_rand_selected_tor_m(duthosts, mux_server_url,
     """
     # Skip on non dualtor testbed or dualtor testbed without active-standby ports
     if 'dualtor' not in tbinfo['topo']['name'] or not active_standby_ports:
+        logger.debug('active_standby_ports: {}'.format(active_standby_ports))
```

Comment:
This line is not needed, right?

Reply:
> This line is not needed, right?

You are right, theoretically then the line prints, `active_standby_ports` should be `None` or empty. 

> Never mind, let's have the fix first.

Thanks for merging this!